corgi/core/models.py

```diff
@@ -408,22 +408,16 @@ def builds(self) -> QuerySet:
     @property
     def cpes(self) -> tuple[str, ...]:
         """Return CPEs for child streams / variants."""
-        # include_self=True so we don't have to override this property for streams
-        descendants = (
-            self.pnodes.get_queryset().get_descendants(include_self=True).using("read_only")
-        )
-        stream_cpes = (
-            descendants.filter(level=MODEL_NODE_LEVEL_MAPPING["ProductStream"])
-            .values_list("productstream__cpe", flat=True)
-            .distinct()
-        )
         variant_cpes = (
-            descendants.filter(level=MODEL_NODE_LEVEL_MAPPING["ProductVariant"])
+            self.pnodes.get_queryset()
+            .get_descendants()
```

Comment:
Let's make sure that if we access `some_variant.cpes`, we'll still get a tuple with one entry like `(some_variant.cpe,)` and not an empty () tuple:
```suggestion
            .get_descendants(include_self=True)
```